### PR TITLE
chore: prop-types card

### PR DESCRIPTION
**Pull request by valselviz**
- Add prop types to game components
- replace ranking with game id
- fix card title style

image

### CODE REVIEW

File: src/components/GamesContainer/GameCard/GameCardInfo/GameCardPlatformIcons/GameCardPlatformIcons.jsx [hunk 1]
```diff
@@ -1,3 +1,4 @@
+import PropTypes from 'prop-types';
 import styles from './GameCardPlatformIcons.module.css';
 
 import icon1 from '../../../../../assets/GameCard/platforms/dark/icon-01.svg';
```
Comment (elios11) [inline src/components/GamesContainer/GameCard/GameCardInfo/GameCardPlatformIcons/GameCardPlatformIcons.jsx]: You could use the `@assets` alias to access the assets folder directly

**Review by elios11 — APPROVED** [review]

File: src/components/GamesContainer/GameCard/GameCardInfo/GameCardInfo.jsx [hunk 2]
```diff
@@ -31,3 +32,7 @@ export default function GameCardInfo({ game, ranking }) {
         </div>
     );
 }
+
+GameCardInfo.propTypes = {
+    game: PropTypes.object,
```
Comment (fgomensoro) [inline src/components/GamesContainer/GameCard/GameCardInfo/GameCardInfo.jsx]: maybe you can use shape object proptype to be more specific. If its too difficult you can leave it as it is. 😬

**Review by fgomensoro — APPROVED** [review]

File: src/components/GamesContainer/GameCard/GameCardInfo/GameCardPlatformIcons/GameCardPlatformIcons.jsx (hunk 1 above)
Comment (valselviz) [reply src/components/GamesContainer/GameCard/GameCardInfo/GameCardPlatformIcons/GameCardPlatformIcons.jsx]: done! thanks

File: src/components/GamesContainer/GameCard/GameCardInfo/GameCardInfo.jsx (hunk 2 above)
Comment (valselviz) [reply src/components/GamesContainer/GameCard/GameCardInfo/GameCardInfo.jsx]: Great, done!

**Pull request merged**